sansa-ml-spark/src/test/scala/net/sansa_stack/ml/spark/clustering/algorithms/LocalitySensitiveHashingTest.scala

```diff
@@ -0,0 +1,26 @@
+package net.sansa_stack.ml.spark.clustering.algorithms
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.apache.jena.riot.Lang
+import org.apache.spark.rdd.RDD
+import org.scalatest.FunSuite
+
+import net.sansa_stack.ml.spark.clustering.algorithms._
+
+class LocalitySensitiveHashingTest extends FunSuite with DataFrameSuiteBase {
+  
+  import net.sansa_stack.rdf.spark.io._
+
+  val path = getClass.getResource("/Cluster/testSample.nt").getPath
+  val graphframeCheckpointPath = "/sansa-ml-spark_2.11/src/main/scala/net/sansa_stack/ml/spark/clustering/algorithms/graphframeCheckpoints"
+  
+  test("performing clustering using HashingTF method") {
+    val triples = spark.rdf(Lang.NTRIPLES)(path)
+
+    val cluster_ = new LocalitySensitiveHashing(spark, triples, graphframeCheckpointPath)
+    cluster_.run()
```

Comment:
Can we get the centroids? and do some validations insted of just run method? and perform `assert(true)` it will always pass right ;) . Check here for inspiration: https://github.com/apache/spark/tree/master/mllib/src/test/scala/org/apache/spark/ml/clustering

Reply:
the `run()` method doesn't return anything, so I'm wondering how somebody would use this code? I  mean, it computes clusters and also their quality, but nothing is returned nor written to disk somewhere ... that can't be useful in its current state.